### ISSUE TITLE
stand up dev mosaic DB

### ISSUE THREAD

**Issue by jeffklassen** [pblair12/mosaic-groups#8]

**Comment by jeffklassen** [comment]
Ill update the dev db to point here. Wont be good for offline development, but will simplify things.

I can also populate the DB. All i need you to do is provision it and send me a link.

**Comment by pblair12** [comment]
done!  

We now have the production/original: 
https://mongolab.com/databases/mosaicgroups

And a copy of that for development
https://mongolab.com/databases/mosaicgroups-dev

I also created a new account user for you.  Go here to log in https://mongolab.com/home your user name is jeffrey.s.klassen and your password is the same password as the mosaicadmin database user.

The adventuresforthecure database is still in this account but I copied it to a different account and my www.adventuresforthecure.com website is now pointing to it.  I will remove that database from this account soon so that only the two mosaic databases will be here.
